contracts/OmnuumNFT1155.sol

```diff
@@ -16,6 +16,7 @@ contract OmnuumNFT1155 is ERC1155Upgradeable, ReentrancyGuardUpgradeable, Ownabl
 
     OmnuumCAManager caManager;
     OmnuumMintManager mintManager;
+    SenderVerifier senderVerifier;
```

Comment:
Is there any reason for adding this variable?

Reply:
Not really. I mixed up.